pkg/agent/openflow/pipeline.go

```diff
@@ -1706,6 +1765,19 @@ func (f *featureNetworkPolicy) conjunctionActionFlow(conjunctionID uint32, table
 			Done()
 	}
 	var flows []binding.Flow
+	// As IGMP and multicast use a different pipeline 'Multicast', if the rule is
+	// IGMP ingress or multicast egress，conjunctionActionFlow generates the flow
+	// to mark the packet to be allowed if policyRuleConjunction ID is matched.
+	// Any matched flow will be resubmitted to next table in corresponding metric tables.
+	if f.enableMulticast && (tableID == MulticastEgressRuleTable.GetID() || tableID == MulticastIGMPIngressTable.GetID()) {
+		flow := table.BuildFlow(ofPriority).MatchConjID(conjunctionID).
+			Action().LoadToRegField(CNPConjIDField, conjunctionID).
+			Action().LoadRegMark(CnpAllowRegmark).Action().GotoTable(table.GetNext()).
```

Comment:
If you only want to goto next table, you can use this action `NextTable()` in flowBuilder

Reply:
done